invenio_rdm_records/resources/config.py

```diff
@@ -63,13 +64,12 @@ def csl_url_args_retriever():
 #
 record_serializers = {
     "application/json": ResponseHandler(JSONSerializer()),
+    "application/marcxml+xml": ResponseHandler(MARCXMLSerializer()),
     "application/vnd.inveniordm.v1+json": ResponseHandler(UIJSONSerializer()),
     "application/vnd.citationstyles.csl+json": ResponseHandler(CSLJSONSerializer()),
-    "application/vnd.datacite.datacite+json": ResponseHandler(
-        DataCite43JSONSerializer()
-    ),
+    "application/vnd.datacite.v43+json": ResponseHandler(DataCite43JSONSerializer()),
     "application/vnd.geo+json": ResponseHandler(GeoJSONSerializer()),
-    "application/vnd.datacite.datacite+xml": ResponseHandler(DataCite43XMLSerializer()),
+    "application/vnd.datacite.v43+xml": ResponseHandler(DataCite43XMLSerializer()),
```

Comment:
@slint should this be:

a) vnd.datacite.datacite+xml (as it was before, no versioning)
b) vnd.datacite.v43+xml
c) vnd.datacite.datacite.v43+xml

I'm leaning towards b (the one implemented), but I'm not sure and I cannot find concrete info. Also, why is this a vendor (vnd) and others like DublinCore are not?

Reply:
Disclaimer, I'm not @slint ;)
Going through Zenodo code it seems that that what is used for v31 is
`application/x-datacite+xml`
See [here](https://github.com/zenodo/zenodo/blob/b27daab5825f310739bc46ca49c9454f65b72ed0/zenodo/modules/records/serializers/__init__.py#L122)

Reply:
I'm not @slint  either (keeping headers for consistency 😂)

Thanks @tmorrell, I think in that case it makes sense to keep `application/vnd.datacite.datacite+xml`. I'm wondering what would happen then if we support both v4.3 and v4.4 or so? 🤔

Reply:
As the real @slint, even though much has already been said, I guess it's my time to comment here :)

Agree that the double `datacite` sounds weird, but maybe there's some reasoning behind it (single doesn't seem work, so it's not a typo in the docs). 

For Zenodo we do have the `application/x-datacite+xml`, which could be from older documentation, but I couldn't honestly track where it came from...

I think at this point specifying the version doesn't make much difference also, since it's just additional complexity, and also the versions are backward compatible (if someone can submit/parse v4.3, the same payload is valid in v4.4). 

So my vote is also for `application/vnd.datacite.datacite+xml`.

Reply:
_Would the real @slint shady please stand up, please stand up_

Thanks for the discussion, I'll revert back to `application/vnd.datacite.datacite+xml` and `application/vnd.datacite.datacite+json`